### PR TITLE
[CAM-10971] feat(tasklist): add 'Assigned' to filter options

### PR DESCRIPTION
**Pull request by daveychu**
related to CAM-10971

### CODE REVIEW

**Comment by tmetzke** [comment]
Hey @daveychu ,

thanks for working on this, we appreciate it!
We will have a closer look and provide proper feedback as soon as we can. :+1:

Thanks and best,
Tobias

**Comment by marstamm** [comment]
Hi [~daveychu],

thank you for your Pull Request. I tested it out and your contribution worked as expected.

Unfortunately your addition revealed a Bug in the REST Api, so we cannot merge it quite yet. I created CAM-12583 to track the progress of the fix for that. If you have any further questions, don't hesitate to ask.

Cheers
Martin

**Review by marstamm — APPROVED** [review]
:heavy_check_mark: 

Note to maintainers: 
**do NOT MERGE before CAM-12583 is resolved**

**Pull request merged**
